package/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM registry.suse.com/bci/bci-base:15.4
```

Comment:
use either bci-minimal or bci-golang since no additional package is required to install